src/main/java/core/basesyntax/repoter/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.repoter;
```

Comment:
The package name 'repoter' seems to be a typo. It should be 'reporter' to correctly reflect the purpose of the classes it contains.

Reply:
fix this